connector/src/main/java/org/geysermc/connector/configuration/GeyserJacksonConfiguration.java

```diff
@@ -38,6 +38,12 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public abstract class GeyserJacksonConfiguration implements GeyserConfiguration {
 
+    /**
+     * If the config was originally 'auto' before the values changed
+     */
+    @Getter @Setter
+    private boolean autoconfiguredRemote = false;
+
```

Comment:
This should be moved to the correct order. It also doesn't need the `@Getter` since that's set at class level.